code/__DEFINES/admin_defines.dm

```diff
@@ -40,11 +39,12 @@
 #define R_PROCCALL		(1<<15)
 #define R_VIEWRUNTIMES	(1<<16)
 #define R_MAINTAINER	(1<<17)
+#define R_DEV_TEAM		(1<<18)
 // Update the following two defines if you add more
 
-#define R_MAXPERMISSION (1<<17) //This holds the maximum value for a permission. It is used in iteration, so keep it updated.
+#define R_MAXPERMISSION (1<<18) // This holds the maximum value for a permission. It is used in iteration, so keep it updated.
 
-#define R_HOST			((1<<18)-1) //17 bit bitmask, update me if we ever add more admin permissions. Sum of all permissions to allow easy setting.
+#define R_HOST			((1<<19)-1) // 18 bit bitmask, update me if we ever add more admin permissions. Sum of all permissions to allow easy setting.
```

Comment:
```suggestion
#define R_HOST			(~0) // Sum of all permissions to allow easy setting.
```
i don`t know, good this solution or not, but it works

Reply:
![delwedd](https://github.com/user-attachments/assets/bced9c72-859c-443d-aa40-46d4b01e1b80)
Seems legit